account_invoice_constraint_chronology/model/account_move.py

```diff
@@ -34,13 +34,18 @@ def _get_older_conflicting_invoices_domain(self):
             ]
         )
 
-    def _raise_older_conflicting_invoices(self):
+    def _raise_older_conflicting_invoices(self, older_move):
         self.ensure_one()
         raise UserError(
             _(
-                "Chronology conflict: A conflicting draft invoice dated before "
-                "{date_invoice} exists, please validate it first."
-            ).format(date_invoice=format_date(self.env, self.invoice_date))
+                "Chronology conflict: A conflicting draft invoice {name} for "
+                "{partner} dated before {date_invoice} exists, please validate it "
+                "first or remove its invoice date."
+            ).format(
```

Comment:
```suggestion
                "Chronology conflict: A conflicting draft invoice %(name)s for "
                "%(partner)s dated before {date_invoice} exists, please validate it "
                "first or remove its invoice date."
```

@rvalyi would you this before merging?

Reply:
Will try to find the time, carnival here :-)

Reply:
Enjoy!